kite-hadoop-dependencies/hdp2-test/pom.xml

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+~ Copyright 2013 Cloudera Inc.
```

Comment:
Maybe we copied this from another file and thus it retains the Cloudera copyright?